joomla/backend-template
# New module login support with language keys

### Summary of Changes
The login screen contains a coustom module "Need Help?".
To localise this module we hav a new module mod_loginsupport. 
It can replace the custom_module "need help?" 

### Testing Instructions
Install the new module via discover.

In the Module Manager:
- Deactivate the module "needHelp".
- Make a new module of type "login support Information"
- You can write a own title or use the parameter "auto_title"
- Choose Position sidebar.

Now the module should be visible as before. 

### Documentation Changes Required
yes. This module must be described

### Additional Information
If this module is accepted, the install/update sql files must be changed.
